OpenRA.Game/ExternalMods.cs

```diff
@@ -50,21 +52,24 @@ public ExternalMods(string launchPath)
 			sheetBuilder = new SheetBuilder(SheetType.BGRA, 256);
 
 			// Load registered mods
-			var supportPath = Platform.ResolvePath(Path.Combine("^", "ModMetadata"));
-			if (!Directory.Exists(supportPath))
-				return;
-
-			foreach (var path in Directory.GetFiles(supportPath, "*.yaml"))
+			foreach (var source in new[] { Platform.SystemSupportDir, Platform.SupportDir })
 			{
-				try
-				{
-					var yaml = MiniYaml.FromStream(File.OpenRead(path), path).First().Value;
-					LoadMod(yaml);
-				}
-				catch (Exception e)
+				var supportPath = Path.Combine(source, "ModMetadata");
+				if (!Directory.Exists(supportPath))
+					return;
+
+				foreach (var path in Directory.GetFiles(supportPath, "*.yaml"))
 				{
-					Log.Write("debug", "Failed to parse mod metadata file '{0}'", path);
-					Log.Write("debug", e.ToString());
+					try
+					{
+						var yaml = MiniYaml.FromStream(File.OpenRead(path), path).First().Value;
+						LoadMod(yaml);
```

Comment:
In case "Support" directory exists, this will be executed twice for the same path (SystemSupportDir = SupportDir).